src/components/pages/ParentContact/ParentContactContainer.js

```diff
@@ -41,8 +41,9 @@ const ParentContactContainer = () => {
               <div className="description">
                 <Row>
                   <p>
-                    If you would like to let us know how we're doing of if you are
-                    experiencing issues, contact us by filling in the form bellow.
+                    If you would like to let us know how we're doing of if you
+                    are experiencing issues, contact us by filling in the form
+                    bellow.
```

Comment:
This is ridiculous, but I'll go ahead and mention...the word "bellow" should be spelled "below" I believe.  Unless we want the user to yell angrily while filling out the form.